apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -248,6 +248,10 @@ export async function updateContact(params: UpdateContactParams): Promise<ILivec
 	return updatedContact;
 }
 
+export async function getContactById(contactId: string): Promise<ILivechatContact | null> {
+	return LivechatContacts.findOneById(contactId);
+}
+
```

Comment:
I wonder if it makes sense to have this function for just calling the model, we could call that directly there, what do you think? Or maybe it will have more logic in the future?

Reply:
There's no more logic. I'll get rid of this function